hugging_face/finetune_vista3d_for_hugging_face_pipeline.ipynb

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    vista3d_hf_model.network.load_state_dict(torch.load(fine_tuned_model_path))
</pre><p> </p><p>Would be better to add <code>weights_only=True</code> in the <code>torch.load</code> to avoid the error after pytorch 2.6.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>